spec/system/add_search_rspec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Search function', type: :system do
+  before do
+    @ruby_programming_language = create :programming_language, name: 'Ruby'
```

Comment:
You should use local variable if it's fine. (Doesn't need to be an instance variable, does it?)

Reply:
@jp-ryuji 👍 👍